packages/affiliates/payouts/2021-04-05_2021-04-12_0022.json

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
this file does not look correctly linted.

Reply:
@evaldofelipe @daywiss can you take a look at this?

Reply:
fixed on this PR for now.